components/brave_wallet_ui/common/hooks/token.ts

```diff
@@ -47,7 +47,7 @@ export default function useTokenInfo (
       }).catch(e => console.log(e))
     }
     setFoundTokenInfoByContractAddress(undefined)
-  }, [tokenContractAddress, visibleTokens, fullTokenList])
+  }, [tokenContractAddress, visibleTokens, fullTokenList, selectedNetwork])
```

Comment:
Perf: update deps to only observe `selectedNetwork.chainId`, since it is the only property of `selectedNetwork` used by this effect